docs/data/data-grid/accessibility/accessibility.md

```diff
@@ -45,6 +45,32 @@ The user can override this setting with the toolbar density selector if provided
 
 The grid responds to keyboard interactions from the user and emits events when key presses happen on the grid cells.
 
+### Tab sequence
+
+According to [WCAG](https://www.w3.org/TR/wai-aria-practices-1.1/#grid), only one of the focusable elements contained by the grid should be included in the page tab sequence.
+For an element to be included in the tab sequence it needs to have a `tab-index` value of zero or greater.
+
+When a cell of the data grid is focused, the first element with the attribute `tab-index=0` will receive the focus.
```

Comment:
Not sure if "inner element" makes sense but I want to say that is the first element INSIDE the focused cell.

```suggestion
When a cell of the grid is focused, the first inner element with `tabIndex=0` will receive the focus.
```